package.json

```diff
@@ -34,6 +34,10 @@
     "webpack-cli": "^4.9.0"
   },
   "dependencies": {
+    "@emotion/react": "^11.5.0",
```

Comment:
So @emotion is another css styling engine very similar to styled components. Currently our webpack is configured to use styled components as the main css engine for MUI. We don't want to mix in two styling engines.  Could you try uninstalling everything @emotion related and install "@mui/styled-engine-sc" instead and see if that works?

Reply:
@jleiandy Thanks for the explanation! I've uninstalled emotion and merged the latest updates from our master branch. The login page still works like before on my end. Let me know if there are any other issues. Thanks!